File: 07/Python3/Presidente.py
#!/usr/bin/python3
# coding: utf-8
"""Classe Presidente """

from Politico import Politico

class Presidente(Politico):
    """ Classe Presidente """

    def __init__(self, nome, partido):
        """ Construtor da classe Presidente """
        Politico.__init__(self)
        self.set_nome(nome)
        self.set_partido(partido)
        self.set_estado("")
        self.set_funcao("administrar bem os impostos federais em prol do País.")

File: Projetos/Políticos/C++/sources/Presidente.cpp
// Presidente.cpp
#include <stdio.h>
#include <stdlib.h>
#include <iostream>
#include "../includes/Presidente.h"
using namespace std;

Presidente::Presidente(string nome,string partido) {
    setNome(nome);
    setSalario(33000);
    setPartido(partido);
    setEstado("");
    setFuncao("administrar bem os impostos federais em prol do País. ");
}
Presidente::~Presidente() {
    setNome("");
    setPartido("");
}
